### PR TITLE
PLF-4217 People suggestion is not removed permanently

### PR DESCRIPTION
**Pull request by trongtt**

### CODE REVIEW

File: component/service/src/main/java/org/exoplatform/social/rest/impl/userrelationship/UsersRelationshipsRestResourcesV1.java [hunk 1]
```diff
@@ -142,10 +142,18 @@ public Response createUsersRelationships(@Context UriInfo uriInfo,
     if (sender == null || receiver == null) {
       throw new WebApplicationException(Response.Status.UNAUTHORIZED);
     }
+
+    Relationship.Type status = null;
+    if (model.getStatus() != null) {
+      try {
+        status = Relationship.Type.valueOf(model.getStatus().toUpperCase());
+      } catch (Exception e) {
+        // Ignore invalid status.
```
Comment (thomasdelhomenie) [inline component/service/src/main/java/org/exoplatform/social/rest/impl/userrelationship/UsersRelationshipsRestResourcesV1.java]: Shouldn't we throw an exception here ? It makes no sense to continue without a valid Relationship status.

Comment (trongtt) [inline component/service/src/main/java/org/exoplatform/social/rest/impl/userrelationship/UsersRelationshipsRestResourcesV1.java]: Actually, I did check the invalid status along with other conditions right after this block code.

Anyway, I'm going to change and throw an exception right here :)

Comment (thomasdelhomenie) [inline component/service/src/main/java/org/exoplatform/social/rest/impl/userrelationship/UsersRelationshipsRestResourcesV1.java]: Ok good, actually it is never good to have an empty catch block.

**Comment by trongtt** [comment]
merged

**Pull request closed without merging**